definitions/SpellVisualEffectName.dbd

```diff
@@ -101,3 +107,4 @@ textureFileDataID<32>
 ribbonQualityID<32>
 dissolveEffectID<32>
 modelPosition<32>
+
```

Comment:
Was this missing before or is it to be considered unintended?

Reply:
Unintended, will go away next rewrite